ignore_tf:
  - "/bl_caster_l_wheel_link"
  - "/bl_caster_rotation_link"
  - "/bl_caster_r_wheel_link"
  - "/base_link"
  - "/br_caster_l_wheel_link"
  - "/br_caster_rotation_link"
  - "/br_caster_r_wheel_link"
  - "/fl_caster_l_wheel_link"
  - "/fl_caster_rotation_link"
  - "/fl_caster_r_wheel_link"
  - "/fr_caster_l_wheel_link"
  - "/fr_caster_rotation_link"
  - "/fr_caster_r_wheel_link"
  - "/head_pan_link"
  - "/torso_lift_link"
  - "/head_tilt_link"
  - "/l_elbow_flex_link"
  - "/l_upper_arm_link"
  - "/l_forearm_roll_link"
  - "/l_gripper_l_finger_tip_frame"
  - "/l_gripper_r_finger_tip_link"
  - "/l_gripper_l_finger_link"
  - "/l_gripper_palm_link"
  - "/l_gripper_l_finger_tip_link"
  - "/l_gripper_motor_screw_link"
  - "/l_gripper_motor_slider_link"
  - "/l_gripper_r_finger_link"
  - "/l_shoulder_lift_link"
  - "/l_shoulder_pan_link"
  - "/l_upper_arm_roll_link"
  - "/l_wrist_flex_link"
  - "/l_forearm_link"
  - "/l_wrist_roll_link"
  - "/laser_tilt_mount_link"
  - "/r_elbow_flex_link"
  - "/r_upper_arm_link"
  - "/r_forearm_roll_link"
  - "/r_gripper_l_finger_tip_frame"
  - "/r_gripper_r_finger_tip_link"
  - "/r_gripper_l_finger_link"
  - "/r_gripper_palm_link"
  - "/r_gripper_l_finger_tip_link"
  - "/r_gripper_motor_screw_link"
  - "/r_gripper_motor_slider_link"
  - "/r_gripper_r_finger_link"
  - "/r_shoulder_lift_link"
  - "/r_shoulder_pan_link"
  - "/r_upper_arm_roll_link"
  - "/r_wrist_flex_link"
  - "/r_forearm_link"
  - "/r_wrist_roll_link"
  - "/torso_lift_motor_screw_link"
  - "/base_bellow_link"
  - "/base_laser_link"
  - "/double_stereo_link"
  - "/sensor_mount_link"
  - "/head_mount_link"
  - "/head_plate_frame"
  - "/head_mount_kinect_ir_link"
  - "/head_mount_kinect_ir_optical_frame"
  - "/head_mount_kinect_rgb_link"
  - "/head_mount_kinect_rgb_optical_frame"
  - "/head_mount_prosilica_link"
  - "/head_mount_prosilica_optical_frame"
  - "/high_def_frame"
  - "/high_def_optical_frame"
  - "/imu_link"
  - "/l_forearm_cam_frame"
  - "/l_forearm_cam_optical_frame"
  - "/l_gripper_led_frame"
  - "/l_gripper_motor_accelerometer_link"
  - "/l_gripper_tool_frame"
  - "/l_torso_lift_side_plate_link"
  - "/laser_tilt_link"
  - "/narrow_stereo_link"
  - "/narrow_stereo_gazebo_l_stereo_camera_frame"
  - "/narrow_stereo_gazebo_l_stereo_camera_optical_frame"
  - "/narrow_stereo_gazebo_r_stereo_camera_frame"
  - "/narrow_stereo_gazebo_r_stereo_camera_optical_frame"
  - "/narrow_stereo_optical_frame"
  - "/openni_depth_optical_frame"
  - "/openni_rgb_optical_frame"
  - "/projector_wg6802418_child_frame"
  - "/projector_wg6802418_frame"
  - "/r_forearm_cam_frame"
  - "/r_forearm_cam_optical_frame"
  - "/r_gripper_led_frame"
  - "/r_gripper_motor_accelerometer_link"
  - "/r_gripper_tool_frame"
  - "/r_torso_lift_side_plate_link"
  - "/wide_stereo_link"
  - "/wide_stereo_gazebo_l_stereo_camera_frame"
  - "/wide_stereo_gazebo_l_stereo_camera_optical_frame"
  - "/wide_stereo_gazebo_r_stereo_camera_frame"
  - "/wide_stereo_gazebo_r_stereo_camera_optical_frame"
  - "/wide_stereo_optical_frame"
  - "/r_forearm_cam_optical_frame_rotated"
  - "/l_forearm_cam_optical_frame_rotated"
